rnd/autogpt_server/Dockerfile

```diff
@@ -24,27 +19,41 @@ ENV POETRY_VERSION=1.8.3 \
     PATH="$POETRY_HOME/bin:$PATH"
 RUN pip3 install poetry
 
-FROM server_base AS server_dependencies
+# Copy and install dependencies
+COPY rnd/autogpt_libs /app/rnd/autogpt_libs
+COPY rnd/autogpt_server/poetry.lock rnd/autogpt_server/pyproject.toml /app/rnd/autogpt_server/
+WORKDIR /app/rnd/autogpt_server
+RUN poetry config virtualenvs.create false \
+    && poetry install --no-interaction --no-ansi
+
+# Generate Prisma client
+COPY rnd/autogpt_server/schema.prisma ./
+RUN poetry config virtualenvs.create false && poetry run prisma generate
```

Comment:
`poetry config virtualenvs.create false` is redundant with line 18 `POETRY_VIRTUALENVS_CREATE=false`

Reply:
I'm testing out adding the env vars to the second image so I can remove these